.github/workflows/build.yml

```diff
@@ -163,11 +167,15 @@ jobs:
     - name: Build Release
       run: |
         cargo build --release
+        
+    - name: Archive Linux artifact
+      run: |
+        tar czvf ./target/release/neovide.tar.gz ./target/release/neovide
 
     - uses: actions/upload-artifact@v1
       with:
-        name: neovide-linux
-        path: ./target/release/neovide
+        name: neovide-linux.tar.gz
+        path: ./target/release/neovide.tar.gz

```

Comment:
Hmm, i don't understand why are this changes here, i just synced branch with master :(

Please ignore this, i'll resolve it later.

Reply:
Linux/windows builds are now distriubuted in a .tar.gz format as changed in https://github.com/Kethku/neovide/pull/822

Reply:
That was becouse target branch was fix-scaling instead of main. Resolved.